volpiano_display_utilities/text_volpiano_alignment.py

```diff
@@ -30,6 +36,9 @@ def _preprocess_volpiano(volpiano_str: str) -> str:
     returns [str]: preprocessed volpiano string
     """
     processed_str = ""
+    # Remove existing clefs
+    volpiano_str = STARTING_MATERIAL.sub("", volpiano_str)
+    volpiano_str = "1---" + volpiano_str
```

Comment:
I'm a fan of creating new variables for things like this, rather than changing strings in place:
```python
volpiano_str_without_clef = STARTING_MATERIAL.sub("", volpiano_str)
volpiano_str_with_clean_clef = "1---" + volpiano_str_without_clef
```
and so on. Maybe those variable names could be better, but you get the idea.